preview.geoshp/ncsa.geo.shp.py

```diff
@@ -134,7 +134,7 @@ def process_message(self, connector, host, secret_key, resource, parameters):
                 self.logger.debug("upload previewer")
 
         except Exception as ex:
-            self.logger.debug(ex.message)
+            self.logger.debug(ex)
```

Comment:
not this round, but we should do something better with this.

Reply:
ok.